tests/associations/belongs-to-many.test.js

```diff
@@ -158,6 +181,83 @@ describe('Associations(BelongsToMany)', function() {
       });
     });
 
+    it('should return associated data by url (3)', function(done) {
+      request.get({
+        url: test.baseUrl + '/hobbies/1/people'
+      }, function(error, response, body) {
+        expect(response.statusCode).to.equal(200);
+        var result = _.isObject(body) ? body : JSON.parse(body);
+        expect(result).to.eql([{
+          "id": 1,
+          "name": "Mr 1",
+          "hobbies": [{
+              "id": 1,
+              "name": "Azerty"
+            }]
+        }]);
+
+        done();
+      });
+    });
+
+    it('should return associated data by url (4)', function(done) {
+      request.get({
+        url: test.baseUrl + '/hobbies/2/people'
+      }, function(error, response, body) {
+        expect(response.statusCode).to.equal(200);
+        var result = _.isObject(body) ? body : JSON.parse(body);
+        expect(result).to.eql([{
+          "id": 1,
+          "name": "Mr 1",
+          "hobbies": [{
+            "id": 2,
+            "name": "Querty"
+          }]
+        },{
+          "id": 2,
+          "name": "Mr 2",
+          "hobbies": [{
+            "id": 2,
+            "name": "Querty"
+          }]
+        }]);
+
+        done();
+      });
+    });
+
+    it('should return associated data by url (5)', function(done) {
+      request.get({
+        url: test.baseUrl + '/things/1'
+      }, function(error, response, body) {
+        expect(response.statusCode).to.equal(200);
+        var result = _.isObject(body) ? body : JSON.parse(body);
+        expect(result).to.be.an('object');
+        expect(result.id).to.be.eql(1);
+        expect(result.people).to.be.an('array');
+        expect(result.people.length).to.be.eql(1);
+        done();
+      });
+    });
+
+    it('should return 404', function(done) {
+      request.get({
+        url: test.baseUrl + '/things/1/people'
+      }, function(error, response, body) {
+        expect(response.statusCode).to.equal(404);
+        done();
+      });
+    });
+
+    it('should return 404', function(done) {
```

Comment:
can you provide more descriptive test names for these two? 'should return 404 for non existent reverse lookup' or something?